File: PHP/areas/square.php
<?php
    class Square extends FigureGeometric {
        private float $side;
        private float $calculate;

        public function __construct(float $side){
            parent::__construct("Square", "Cuadrado");
            $this->side = $side;
        }
        
        public function getSide(): float{
            return $this->side;
        }

        public function calculate(): float {
            $this->calculate = $this->getSide() * $this->getSide();
            return $this->calculate;
        }
    }
?>

File: Java/figureGeometric/Square.java
public class Square extends FigureGeometric{
    private final double side;
    private double calculate;
 
    public Square(double side){
        super("Square", "cuadrado");
        this.side=side;
    }

    public double getSide(){
        return this.side;
    }
    @Override
    public double calculateArea(){
        this.calculate=this.getSide() * this.getSide();
        return this.calculate;
    }
}
